metricbeat/module/aerospike/mtest/runner.go

```diff
@@ -15,22 +15,26 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package apache
+// +build integration
+
+package mtest
 
 import (
-	"os"
+	"github.com/elastic/beats/libbeat/tests/compose"
 )
 
-// Helper functions for testing the Apache module's MetricSets.
-
-// GetApacheEnvHost returns the apache server hostname to use for testing. It
-// reads the value from the APACHE_HOST environment variable and returns
-// 127.0.0.1 if it is not set.
-func GetApacheEnvHost() string {
-	host := os.Getenv("APACHE_HOST")
-
-	if len(host) == 0 {
-		host = "127.0.0.1"
+var (
+	// Runner is the compose test runner for aerospike
+	Runner = compose.TestRunner{
+		Service: "aerospike",
+		Options: compose.RunnerOptions{
```

Comment:
The config files I started to introduce here could be a place where we have this documented in the future: https://github.com/elastic/beats/pull/11131/files#diff-7d1232f25fa4ea3006884196c00228c7R3